js/configs.js

```diff
@@ -164,30 +164,30 @@ class Main extends Component {
           config: key,
           value: value,
           defaultValue: defaultValue,
-        }
+        };
 
         if (typeof defaultValue === 'boolean') {
           props.onChange = (e) => {
             if (!userConfigs[group]) {
               userConfigs[group] = {};
             }
-            const checked = e.target.checked
+            const checked = e.target.checked;
             if (checked === defaultValue) {
               delete userConfigs[group][props.config];
             } else {
               userConfigs[group][props.config] = checked;
             }
 
             this.setState({
-              userConfigs: userConfigs,
+              userConfigs: userConfigs
```

Comment:
@J2TeaM please [keep all trailing commas](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8)